lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextLifeCycle.cs

```diff
@@ -71,10 +71,10 @@ public void Start()
             IContextStart contextStart = new ContextStartImpl(Id);
             
             // TODO: enable
-            //foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
-            //{
-            //    startHandler.OnNext(contextStart);
-            //}
+            //// foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
```

Comment:
Actually, "in which case the leading space can be omitted."
If that looks weird to you, may I change all leading space for that case?